docs/docs/monitoring/analytics/realtime-markers.mdx

```diff
@@ -0,0 +1,87 @@
+---
+id: realtime-markers
+sidebar_label: Real-Time Markers
+title: Real-Time Analysis of Markers
+description:
+abstract: Process Markers real-time from Rasa
+    Assistants to track metrics like solution and abandonment rates,
+    add metatags to conversations,
+    or filter conversations for Conversation Driven Development
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import RasaProLabel from "@theme/RasaProLabel";
+import RasaProBanner from "@theme/RasaProBanner";
+import { Grid } from "@theme/Grid";
+
+<RasaProLabel />
+
+:::info Available in Rasa Pro from 3.6.0
+:::
+
+<RasaProBanner />
+
+In Rasa Pro, [Markers](../../markers.mdx) is a powerful feature that allows
+you to track and extract custom conversational events. With the Analytics
+Data Pipeline, you can now process these Markers in real-time, enabling
+you to gain valuable insights and enhance the performance of your Rasa
+Assistant. In this guide, we'll explore how to leverage real-time
+analysis of Markers to track solution rate and abandonment in your conversations.
+
+## Defining Markers
+
+Please consult the [Markers](../../markers.mdx/#defining-markers) section of Rasa documentation
+for details about defining Markers.
+
+## Enable Real-time Processing
+
+To enable real-time analysis of Markers in your Rasa project, you need to
+upload the markers to Analytics Data Pipeline with the `rasa markers upload`
+command. This command uploads the marker configuration YAML and starts the
+processing of Markers in the Analytics Data Pipeline. To get started,
+make sure you have the latest version of Rasa installed. Then, open your command line
+interface and navigate to your Rasa project directory. Run the following command:
+
+```
+rasa markers upload --config=<path-to-config-file> -d=<path-to-domain-file> -rasa-pro-services-url=<url>
+```
+
+By default, this command validates the marker configuration file against
+your bot's domain.yml file. To specify a different domain file, use the
+optional `-d` argument.
+
+This command should be run whenever there is a change in the marker
+configuration file. The changes might include addition of new markers,
+changing an existing marker or removing an existing marker.
+
+:::note
+
+This command **uploads** the marker configurations to the data pipeline. The pipeline assumes the configuration
+file is the source of truth and only processes the markers defined in it. If you remove a marker from this file,
+and run this command, then the processing for that marker is stopped.
+
+:::
+
+### Configuring the CLI command
+
+Visit our [CLI page](./command-line-interface.mdx#rasa-marker-upload)
+for more information about the command line arguments available.
+
+## How are Markers processed?
+
+Once the marker definitions are uploaded to the Analytics Data Pipeline, they
```

Comment:
What's a "marker definition"? Above, it says "This command uploads the marker configurations to the data pipeline." and here it's talking about the "marker definitions are uploaded"?

Reply:
I agree that's a bit confusing. Marker definition or marker configuration refers here to the YAML files that describe the marker in the following manner,

```
marker_cheer_up_success:
  description: "Sequence of intent and action that define the marker_cheer_up_success"
  seq:
    - intent: mood_unhappy
    - action: utter_cheer_up
    - action: utter_did_that_help
    - intent: affirm
```

This information is used to identify when these event sequences occur in the conversation. Or, when `marker_cheer_up_success` is extracted from a conversation. To differentiate these, I have referred the YAML files as "marker-configuration" and the markers found in the conversations as "extracted markers". 

Do you have any suggestions for making this distinction more clear and simplified?